app/robot-shop/helm/templates/rabbitmq-deployment.yaml

```diff
@@ -51,4 +51,73 @@ spec:
             {{- with .Values.rabbitmq.nodeSelector }}
             nodeSelector:
               {{- toYaml . | nindent 14 }}
-            {{- end }}
\ No newline at end of file
+            {{- end }}
+---
```

Comment:
Let's create a separate manifest for monitors. This one is specifically namespaced for deployment.